spec_util/ir_hash/gen/gen.go

```diff
@@ -32,13 +32,17 @@ func main() {
 	gf.AddIgnoredField("HTTPMethodMeta", "ProcessingLatency")
 	gf.AddIgnoredField("Data", "ExampleValues")
 
+	// Newer protobuf compiler uses these istead of XXX prefix
```

Comment:
Nit: Might be nice to put the old vs. new versions in the comment, e.g. `Newer protobuf compiler (1.39) uses these instead of XXX prefix (1.26)`